image/hack/get_ci_vm.sh

```diff
@@ -30,7 +30,7 @@ if [[ "$1" == "--config" ]]; then
     in_get_ci_vm "$1"
     cat <<EOF
 DESTDIR="/var/tmp/go/src/github.com/containers/image"
-UPSTREAM_REPO="https://github.com/containers/image.git"
+UPSTREAM_REPO="https://github.com/containers/container-libs.git"
```

Comment:
That is most likely not enough but I guess it won't do any harm either so that is fine